numpy/lib/tests/test_npyio.py

```diff
@@ -0,0 +1,53 @@
+from numpy.testing import assert_, assert_raises
```

Comment:
That was unclear - I meant that I think this whole file should just be an addition to `test_io.py` Are you sure that that tests a different module? If so, which one?